docs/local_install_skaffold.md

```diff
@@ -1,51 +1,280 @@
+    - [Minikube useful commands](#minikube-useful-commands)
+    - [Tiller](#tiller)
+    - [Ongoing issues](#ongoing-issues)
+  - [7. Further resources](#7-further-resources)
+  - [8. Need help](#8-need-help)
+  - [9. Acknowledgements](#9-acknowledgements)
+___
+
+When everything is ready, you will be able to start Substra with:
+```sh
+minikube start --cpus 6 --memory 8192 --disk-size 50g --kubernetes-version='v1.15.4'
+
+skaffold dev # in hlf-k8s, susbtra-backend, substra-frontend repositories
+```
+
+## 1. Requirements
+
+### General knowledge
+
+I order to install Substra, it is *recommended* to be confortable with your package manager and to have some basic knowledge of Linux administration and network.
+
+### Hardware requirements
+
+> Please be cautious with the parameters passed to Kubernetes as it might try to use all the allocated resources without regards for your system!
+
+If you wish to comfortably run Substra, it is advised to have:
+
+- At least 30 Go of free space but **50** would really be better!
+- **8 Go of RAM** for Kubernetes/Minikube! Be warned, 4 Go won't work. 
+
+### Software requirements
+
+> Note: Please **always** refer to the package provider website before installing any software!
+
+> Note: This setup has been tested on Ubuntu 19.10 & 18.04.3, Linux Mint 19 Cinnamon (4.15.0-45-generic Kernel)
+
+Substra deployment is orchestrated by `Kubernetes` and `Minikube` is a great tool for your local Kubernetes deployments.
+
+`Helm` is a Kubernetes package manager that helps install and manage Kubernetes applications. `Tiller` runs inside the Kubernetes Cluster and manages releases of your charts. Helm has two parts : helm (client) & tiller (server). Charts are Helm packages that contains a package description (`Chart.yaml`) and one or more templates which contain Kubernetes manifest files. Charts can be stored locally or on a distant repository.
+
+`Skaffold` is a program working on top of the Kubernetes configuration that will operate the deployment for you. It relies on the `skaffold.yaml` files that you will find at the root of each repositories.
+
+#### [1. Kubernetes](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+##### Mac
+
+First of all, download the `Docker desktop` installer from https://www.docker.com/products/docker-desktop. You'll have to create an account there to do so. Then run it to install Docker on your machine.
 
-This guide will help you run the Substra platform on your machine in development mode, with a 2 organisations setup.
+Once installed, launch Docker and open its "preferences" panel. In the Kubernetes tab, check the `Enable Kubernetes` checkbox.
+
+Kubernetes will take a while to launch the first time, but once it is done, you can move on to configuring.
+
+If you don't have `homebrew` installed, please install it first with the following command:
+
+```sh
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
```

Comment:
Instead of duplicating the install instructions from homebrew's website, why not send the reader there ?

Reply:
Sure!